plugins/filter_kubernetes/kubernetes.c

```diff
@@ -847,6 +847,11 @@ static struct flb_config_map config_map[] = {
      0, FLB_TRUE, offsetof(struct flb_kube, kubelet_port),
      "kubelet port to connect with when using kubelet"
     },
+    {
+     FLB_CONFIG_MAP_INT, "kube_token_ttl", "600",
```

Comment:
this should be `FLB_CONFIG_MAP_TIME`, then you can use `10m` as the default

Reply:
If we do this, the field on the struct should be `uint64` ?

Reply:
Never mind, `flb_utils_time_to_seconds` returns an int. (which I think means the time config map type won't work well on systems where an integer is smaller than 32 bits, since a 16 bit integer can only hold enough seconds for about 1 day if I did my math right)